### ISSUE TITLE
change tool name ? 

### ISSUE THREAD

**Issue by 12rambau** [gee-community/eetasks#26]
I saw your latest post on linkedIn, and I was super impressed, I will definitely try the mapping tool. 
Considering that you are extending the cover of the code editor in VS code don't you think your tool should be reanmed to reflect it ? 

something like eeVSCode or equivalent ?

**Comment by lopezvoliver** [comment]
I agree and I'm open to suggestions for changing the `displayName` (currently "Earth Engine Task Manager"). I don't think the actual `name` can be modified because it's used as part of the unique identifier for the extension ($publisher.$name). 

The `description` can also be updated and is currently set to "Monitor and submit Google Earth Engine (GEE) tasks."

https://code.visualstudio.com/api/references/extension-manifest#marketplace-presentation-tips

**Comment by 12rambau** [comment]
it's forbidden to use the gee logo, I tried to set it once for this community and the Google team requested it to be removed within a week. 
On the other hand many of the repositories of this organisation have customised version of the logo: 
- geemap: https://github.com/gee-community/geemap/blob/master/docs/assets/logo.png
- geetools: https://raw.githubusercontent.com/gee-community/geetools/main/docs/_static/logo.svg
- pytest-gee: https://raw.githubusercontent.com/gee-community/pytest-gee/main/docs/_static/logo.svg
- ipygee (that is essentially the same as what you do but for jupyterlab): https://github.com/gee-community/ipygee
- ee-palette: https://github.com/gee-community/ee-palettes/blob/master/misc/ee-palettes-icon.png
yours could be mix between the vscode and earthengine one.
